src/streaming/MediaPlayer.js

```diff
@@ -1106,6 +1107,39 @@ function MediaPlayer() {
     }
 
     /**
+     * Sets the ABR strategy. Valid strategies are "abrDynamic", "abrBola" and "abrThroughput".
+     * The ABR strategy can also be changed during a streaming session.
+     * The call has no effect if an invalid method is passed.
+     *
+     * The BOLA strategy chooses bitrate based on current buffer level, with higher bitrates for higher buffer levels.
+     * The Throughput strategy chooses bitrate based on the recent throughput history.
+     * The Dynamic strategy switches smoothly between BOLA and Throughput in real time, playing to the strengths of both.
+     *
+     * @param {string} value
+     * @default "abrDynamic"
+     * @memberof module:MediaPlayer
+     * @instance
+     */
+    function setABRStrategy(value) {
+        if (value === Constants.ABR_STRATEGY_DYNAMIC || value === Constants.ABR_STRATEGY_BOLA || value === Constants.ABR_STRATEGY_THROUGHPUT) {
```

Comment:
Is it time to create a public facing TYPES object accessible from outside Dash.js?